src/org/jitsi/impl/neomedia/transform/dtls/DtlsControlImpl.java

```diff
@@ -397,7 +522,7 @@ private static AsymmetricCipherKeyPair generateKeyPair()
         {
             long now = System.currentTimeMillis();
             Date notBefore = new Date(now - ONE_DAY);
-            Date notAfter = new Date(now + 6 * ONE_DAY);
+            Date notAfter = new Date(now + CERT_VALIDITY_TIME);
```

Comment:
I suppose for the same reason as (now - ONE_DAY)?
Which I am guessing is to avoid timezone or clock setting issues, so I can see this one.

But if someone bothers to change it, why not give them the ability to choose?